### ISSUE TITLE
Apply database scheme to database

### ISSUE THREAD

**Issue by jandegen** [amosproj/amos2023ss04-kubernetes-inventory-taker#65]
## User story
1. As a developer
2. I want / need the database scheme applied to the database
3. So that I can write data into it

## Acceptance criteria
- [ ] Database scheme is applied

## Linked Issues
- **Blocked by:**
- **Related to:**
- **Implements** #46 

## Additional information
- none

## Definition of done (DoD)
* Colours match with [color scheme](https://amos-p4.atlassian.net/l/cp/g5VzjL88)

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
